css/_vertical_filmstrip_overrides.scss

```diff
@@ -159,4 +156,17 @@
             transition-delay: 0.1s;
         }
     }
+
+    /**
+     * Apply hardware acceleration to prevent flickering on scroll. The
+     * selectors are specific to icon wrappers to prevent fixed position dialogs
+     * and tooltips from getting a new location context due to translate3d.
+     */
+    .connection-indicator,
+    .remote-video-menu-trigger,
+    .videocontainer__toolbar,
+    .raisehandindicator,
+    #dominantspeakerindicator {
+        transform: translate3d(0, 0, 0);
+    }
```

Comment:
position:fixed inline dialog will position itself relative to a parent that has translate3d. The icons are inside __toolbar and __toptoolbar, so leaving translate3d on them would cause the inline dialogs to look position:absolute. So, move the hardware acceleration closer to the icons that need them so none of the parents have position:fixed.